src/src/components/form/CustomButton.vue

```diff
@@ -0,0 +1,36 @@
+<template>
+      <button
```

Comment:
Codacy has a fix for the issue: [Expected indentation of 2 spaces but found 6 spaces.](https://app.codacy.com/gh/Supershift/Sushi.Mediakiwi.UI/pullRequest?prid=8211123)
```suggestion
  <button
```